templates/generic/action_ajax.html

```diff
@@ -21,3 +21,17 @@
   <i class="{{ icon_class }} htmx-hide"></i>
   <span class="loading loading-spinner loading-xs htmx-show"></span>
 </button>
+<script>
+  document.body.addEventListener('htmx:responseError', function(evt) {
+    alertify
+      .alert(evt.detail.xhr.responseText, function(){
+        alertify.message('OK');
+      }).setting({
+        title: 'Error',
+        transition: 'fade',
+        onok: function() {},
+      }).show();
+    ;
+
+  });
+</script>
```

Comment:
maybe we should make this a global handler on the site?

Reply:
https://github.com/dimagi/open-chat-studio/pull/810/commits/39c9364be6567c802ac736a59000a309fdf5dd32